src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking.cs

```diff
@@ -3779,6 +3779,9 @@ struct S2
                 // (133,18): warning CS8600: Cannot convert null to non-nullable reference.
                 //         y15.F3 = null;
                 Diagnostic(ErrorCode.WRN_NullAsNonNullable, "null").WithLocation(133, 18),
+                // (135,15): warning CS8601: Possible null reference assignment.
```

Comment:
I didn't understand why this wasn't reported before. `u15` is a local with reference type and `y15.F3` is a field with reference type inside a class.
Was that just a bug?